### PR TITLE
Update Get-CsOnlineUser.md

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -252,31 +252,25 @@ _New Operators_
 
 These filtering operators have been reintroduced:
 
-- "-like" operator now supports the use of wildcard operators in 'contains' and 'ends with' scenarios. For example:
+"-like" operator now supports the use of wildcard operators in 'contains' and 'ends with' scenarios. For example:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:255]:
```suggestion
`-like` operator now supports the use of wildcard operators in 'contains' and 'ends with' scenarios. For example:
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -252,31 +252,25 @@ _New Operators_
...
-```
-Contains Scenario: Get-CsOnlineUser  -Filter "DisplayName -like '*abc*'"
-Ends with scenario: Get-CsOnlineUser  -Filter {DisplayName -like '*abc'}
-```
+- Contains Scenario: Get-CsOnlineUser  -Filter "DisplayName -like '*abc*'"
+- Ends with scenario: Get-CsOnlineUser  -Filter {DisplayName -like '*abc'}
 
-- "-contains" can now be used to filter properties that are an array of strings like FeatureTypes, ProxyAddresses, and ShadowProxyAddresses. For example:
-```
+"-contains" can now be used to filter properties that are an array of strings like FeatureTypes, ProxyAddresses, and ShadowProxyAddresses. For example:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:260]:
```suggestion
`-contains` can now be used to filter properties that are an array of strings like FeatureTypes, ProxyAddresses, and ShadowProxyAddresses. For example:
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 3]
```diff
@@ -252,31 +252,25 @@ _New Operators_
...
-```
+- Get-CsOnlineUser -Filter {FeatureTypes -contains "PhoneSystem"}
+- Get-CsOnlineUser -Filter {ProxyAddresses -contains "SMTP:abc@xyz.com"}
 
-- "-gt" (greater than), "-lt" (less than), "-le" (less than or equal to) can now be used for filtering all string properties. For example:
 
-```
-Get-CsOnlineUser -Filter {UserPrincipalName -gt/-le/-lt "abc"}
-```
+"-gt" (greater than), "-lt" (less than), "-le" (less than or equal to) can now be used for filtering all string properties. For example:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:266]:
```suggestion
`-gt` (greater than), `-lt` (less than), and `-le` (less than or equal to) can now be used for filtering all string properties. For example:
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 4]
```diff
@@ -252,31 +252,25 @@ _New Operators_
...
 
-- "-ge" (greater than or equal to) can now also be used for filtering on policies. For example:
+- Get-CsOnlineUser -Filter {UserPrincipalName -gt/-le/-lt "abc"}
+
+"-ge" (greater than or equal to) can now also be used for filtering on policies. For example:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:270]:
```suggestion
`-ge` (greater than or equal to) can now also be used for filtering on policies. For example:
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @isrumnon, one more, please.

**Pull request merged**